Makefile

```diff
@@ -56,15 +62,16 @@ all: build test
 build: LDFLAGS += -X github.com/cockroachdb/cockroach/util.buildTag "$(shell git describe --dirty)"
 build: LDFLAGS += -X github.com/cockroachdb/cockroach/util.buildTime "$(shell date -u '+%Y/%m/%d %H:%M:%S')"
 build: LDFLAGS += -X github.com/cockroachdb/cockroach/util.buildDeps "$(shell GOPATH=${GOPATH} build/depvers.sh)"
+build: TAGS += release
```

Comment:
Unfortunate but, apparently, true. I added the release tag mostly because we now omit the embedded certs when it's set. Now most of the time it won't be doing anything (it will of course on a fresh build).
Is it worth leaving it in? Should we have a `make release` which is just `make build` with the tag?